mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -76,6 +78,8 @@
 
     @Rule
     public ExpectedException expected = ExpectedException.none();
+    @Rule
+    public ErrorCollector collector = new ErrorCollector();
```

Comment:
I tried to verify some functions should be called but I couldn't. Sorry that I forgot to remove it